commons/test/config_loader_test.py

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright 2012-2016, the original author or authors.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance
+# with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+from unittest import TestCase
+
+class ConfigLoaderTest(TestCase):
+    def test_fetch(self):
+        pass
```

Comment:
Sorry for my lack of ignorance. But is this test and below one really testing anything ? Or just passing always ?